routes/[version]/apikey.js

```diff
@@ -0,0 +1,55 @@
+const { checkAuth } = require('../../middlewares/auth');
+const { selectAPIToken, setAPIToken, removeAPIToken } = require('../../services/db');
+
+module.exports = {
+    get: [checkAuth, get],
+    put: [checkAuth, put],
+    delete: [checkAuth, del],
+};
+
+function generateAPIToken() {
```

Comment:
`Math.random()` is not ok for cryptography. `crypto` is ok. I propose this alternative:

```js
const generateAPIToken = (length = 64) => 'METIS' + crypto.randomBytes(length).toString("base64");
```

Reply:
@blokhin don't reinvent the wheel
**uuid** is enough or use another library :)

Reply:
Yes, `crypto.randomUUID()` is nice too. And available without external dependency.